doc/sphinx-guides/source/_static/installation/files/root/big-data-support/checksumValidationSuccess.json

```diff
@@ -1,5 +1,5 @@
 {
   "status": "validation passed",
-  "uploadFolder": "DNXV2H",
-  "totalSize": 1234567890
+  "uploadFolder": "OS7O8Y",
+  "totalSize": 72
```

Comment:
What is this change needed for?

Reply:
oh, this is rsync related stuff. the file there is used for the mock dcm setup. And I had to change the dataset identifier to point to the one I'm currently using. This can be ignored. I added it by mistake!

Reply:
@rbhatta99 @ferrys OK if we move to QA or does this need to be removed?

Reply:
This will not matter for QA at all, as this is just for DCM purposes. But, don't out this into QA just yet. I got some feedback from @sekmiller and I'm trying something out.